src/main/java/Main.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Arrays;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Excavator(), new Bulldozer()};
+        Arrays.stream(machines).forEach(machine -> {
+            machine.doWork();
+            machine.stopWork();
+        });
```

Comment:
such an advanced construction - great job

not critical so you can leave it like it is now, but streams are better for doing other operations
for a simple operation like this - regular for-each is a perfect solution (it is not always a great idea to complicate things)